src/components/Dashboard/widgets/Embeds/ActivityListEmbed.tsx

```diff
@@ -128,29 +143,34 @@ const ActivityList = ({ transactions = [], resourceType }: ActivityListProps): R
             : ActivityType.MINTED;
           const activityData = ActivityMap[activityType];
           return (
-            <Transaction key={tx.txid + tx.received} onClick={() => handleTxDetailView(tx)}>
-              <TriCell
-                icon={tx.unconfirmed ? clockIcon : checkIcon}
-                primary={times[0]}
-                secondary={times[1]}
-              />
-              <TriCell
-                icon={activityData.icon}
-                primary={activityData.primary}
-                secondary={activityData.secondary}
-              />
-              <TriCell
-                primary={` ${tx.received ? '+' : '-'}${toBitcoinAmount(
-                  processPossibleBN(tx.value)
-                )} ${TICKER}`}
-                secondary={
-                  resourceType === ResourceType.TOKEL
-                    ? `$${getUsdValue(processPossibleBN(tx.value), tokelPriceUSD)}`
-                    : ''
-                }
-                justify="flex-end"
-              />
-            </Transaction>
+            <ActivityListItem key={tx.txid + tx.received}>
```

Comment:
we can swap this key to be just the txid

```suggestion
            <ActivityListItem key={tx.txid}>
```